.github/workflows/nightly_testing.yml

```diff
@@ -39,7 +39,7 @@ jobs:
     - name: Setup Spec Test Matrix
       id: set-matrix
       run: |
-        echo "matrix={\"platforms\":[\"rhel-8\",\"rhel-9\",\"ubuntu-2204-lts\"]}" >> $GITHUB_OUTPUT
+        echo "matrix={\"platforms\":[\"rhel-7\",\"rhel-8\",\"rhel-9\"]}" >> $GITHUB_OUTPUT
```

Comment:
why no more ubuntu?

Reply:
Good catch! I'll add a note up top but this was something that was missed when we pulled Ubuntu from the test matrix because the default image doesn't have enough storage space to install and configure both Puppet and Splunk (PIE-1608).